tests/test_satimg.py

```diff
@@ -23,6 +22,34 @@ def test_load_subclass(self):
         img = si.SatelliteImage(fn_img, read_from_fn=False)
         img = si.SatelliteImage(fn_img)
 
+    def test_copy(self):
+
+        fn_img = datasets.get_path("landsat_B4")
+
+        img = si.SatelliteImage(fn_img)
+
+        img2 = copy.copy(img)
+        img3 = img.copy()
+
+        # check the copied object is indeed a SatelliteImage
+        assert isinstance(img2,geoutils.satimg.SatelliteImage)
+        assert isinstance(img3,geoutils.satimg.SatelliteImage)
+
+        # check all immutable attributes are equal
+        georaster_attrs = ['bounds', 'count', 'crs', 'dtypes', 'height', 'indexes', 'nodata',
```

Comment:
There's a list of these attributes at the top of georaster, so better to use that one (in case it's updated)

Reply:
You mean calling it from georaster?

Reply:
Done. Calling all attributes from the class from Raster, SatelliteImage and DEM. Some have to be removed in the test for Raster.